wfe/wfe.go

```diff
@@ -1436,7 +1443,9 @@ func (wfe *WebFrontEndImpl) FinalizeOrder(
 	orderAccountID := existingOrder.AccountID
 	orderStatus := existingOrder.Status
 	orderExpires := existingOrder.ExpiresDate
-	orderNames := existingOrder.Names
+	// not using ordernames now, can't split dns and ips
+	//	orderNames := existingOrder.Names
```

Comment:
Please don't leave commented code in-file. It should be removed entirely or left as-is. In this case I think it should stay. I think this change will introduce a regression that would allow an ACME client to create an order with duplicated identifiers. Can you expand on why you weren't able to make this work with IP address identifiers, I'm not sure I understand based on the comment text.